src/Uno.UI/UI/Xaml/Controls/Image/Image.cs

```diff
@@ -421,12 +421,15 @@ protected override Size ArrangeOverride(Size finalSize)
 					}
 				}
 
+#if __ANDROID__
+				// Images on UWP are always clipped to the control's boundaries.
+				var physicalSize = finalSize.LogicalToPhysicalPixels();
+				ImageControl.ClipBounds = new Android.Graphics.Rect(0, 0, (int)physicalSize.Width, (int)physicalSize.Height);
```

Comment:
Always floored ?

Reply:
Yep. I confirmed that. UWP is doing the same thing, by the way. Since it's physical pixels, you can't cut them in half ;-)